packages/@aws-cdk/aws-ecs/test/fargate/fargate-task-definition.test.ts

```diff
@@ -44,7 +45,7 @@ describe('fargate task definition', () => {
       // GIVEN
       const stack = new cdk.Stack();
       const taskDefinition = new ecs.FargateTaskDefinition(stack, 'FargateTaskDef', {
-        cpu: 128,
+        cpu: 1024,
```

Comment:
Can you add the new property to this test? the test is called 'with all properties set'